frontend/config/agents.ts

```diff
@@ -61,13 +63,11 @@ export const AGENT_CONFIG: {
     evmHomeChainId: EvmChainId.Gnosis,
     middlewareHomeChainId: MiddlewareChainMap.GNOSIS,
     agentIds: [14, 25],
-    requiresAgentSafesOn: [EvmChainId.Gnosis],
-    requiresMasterSafesOn: [EvmChainId.Gnosis],
+    defaultStakingProgramId: STAKING_PROGRAM_IDS.PearlBeta,
```

Comment:
I've faced many issues with that we keep default staking programs by chain not by agent, while the second is more correct. removed it from constants and added to config instead